src/heartratetask/HeartRateTask.h

```diff
@@ -26,6 +26,7 @@ namespace Pinetime {
       static void Process(void* instance);
       void StartMeasurement();
       void StopMeasurement();
+      __ULong total;
```

Comment:
Prefer standard types (`uint32_t` I guess ?)

Reply:
Thanks for the feedback! Note that because of the undefined behavior of the sensor (mainly because of the incomplete and confusing HRS3300 datasheet) the "magic" values above are experimental adjustments modifying the values that you ported over from Daniel Thomson's code. I did wonder if I should retain old comments or not because of the sensor remaining in a state of perpetual reverse engineering! Makes sense that Git will track it all.
The current data you collected is of interest to me. Is it possible for you send me the data set with the acquisition frequency or timestamps? I'm trying to understand why and how the driver oscillates after changing the low nibble in PDriver (0xE or 0xF) with wait time set to zero. The current oscillations look like a good way to measure that!

Reply:
Please post a new comment instead of replying to the thread about using __ULong. Code review threads will be hidden once resolved.

Reply:
Thanks, missed this one. I will address in the next commit.

Reply:
Removed algorithm time measurement code so "_ULong total" no longer exists.